diff --git a/node_modules/@adonisjs/lucid/build/src/Migrator/index.js b/node_modules/@adonisjs/lucid/build/src/Migrator/index.js
index c3f9821..7811b19 100644
--- a/node_modules/@adonisjs/lucid/build/src/Migrator/index.js
+++ b/node_modules/@adonisjs/lucid/build/src/Migrator/index.js
@@ -330,6 +330,7 @@ class Migrator extends events_1.EventEmitter {
          */
         this.emit('create:schema_versions:table');
         await this.client.schema.createTable(this.schemaVersionsTableName, (table) => {
+            table.increments().notNullable();
             table.integer('version').notNullable();
         });
     }
